dsctriage/dscfinder.py

```diff
@@ -196,11 +196,11 @@ def create_editor_name_str(post, site=None):
                     author_name = json_output["username"]
 
                     user_url = create_url(USER_JSON_URL, json_output["username"])
-                    with request.urlopen(user_url) as user_url_data:
-                        user_json_output = json.loads(user_url_data.read().decode())
+            with request.urlopen(user_url) as user_url_data:
```

Comment:
Can you add a check here to make sure user_url is not None, otherwise the program will crash when a username is not found

Reply:
Ah, sorry missed that - i've broadened the change so that the request is closed as soon as we have JSON loaded, which then avoids this problem

Reply:
Thanks, I agree this looks a lot better and helps keep the requests from being active unnecessarily